INSTALL.md

```diff
@@ -37,8 +37,8 @@ $ conda install -c pytorch/label/nightly faiss-cpu
 # GPU(+CPU) version
 $ conda install -c pytorch/label/nightly -c nvidia faiss-gpu=1.9.0
 
-# GPU(+CPU) version with NVIDIA RAFT
-conda install -c pytorch -c nvidia -c rapidsai -c conda-forge faiss-gpu-raft=1.9.0 pytorch pytorch-cuda numpy
```

Comment:
`conda install -c pytorch -c rapidsai -c conda-forge pytorch/label/nightly::faiss-gpu-cuvs pytorch pytorch-cuda numpy` works for me.

Reply:
The raft nightly would no longer exist. So I've removed this. Instead we have install instructions for the cuvs nightly.

Reply:
The proposed command in your diff installs an older version of the nightly for me compared to the command I provided in my previous comment.

Reply:
It seems to be installing libcuvs=24.12 for me. Prefixing the package and using the same channel label as a channel should be equivalent. Perhaps we are doing something differently. I am using this command:
`conda install -c pytorch/label/nightly -c rapidsai -c conda-forge -c nvidia faiss-gpu-cuvs 'cuda-version>=12.0,<=12.5'`

Reply:
Updated this!